rest/src/routes/accountRoutes.js

```diff
@@ -74,5 +78,20 @@ module.exports = {
 						.then(transactionSender.sendArray('accountId', res, next)));
 			});
 		});
+
+		server.get('/account/:accountId/transactions/incoming', (req, res, next) => {
```

Comment:
i believe we can keep the old logic and just pass an address, we can rename the `accountAddress`/`publicKey` to `account` or similar to generalize